### PR TITLE
Millstone 3: Desktop first

### PR DESCRIPTION
**Pull request by cherelemma**
In this project work, I built a **responsive desktop version portfolio** using HTML, CSS, and media query in this project. More specifically, I 

- Setup desktop version portfolio
- Revise/update mobile first HTML file.
- Add all the portfolio sections.
- Create CSS using media query.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,27 +4,38 @@
...
+      <section class="headline-container">
+        <article class="headline">
+          <h1 class="primary-text">Hey There. <br> I&rsquo;m Chere</h1>
+          <h2 class="support-text">I&rsquo;am a Software Developer</h2>
+    
+          <p class="description">I can help you build a product, feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, <br> don’t hestiate to contact me.</p>
+        </article>
         
         <ul class="social-icons">
```
Comment (roshan-bajgain) [inline index.html:40]: - [ ] You should stick to the design as much as possible (e.g., font, colors, images, text, margins). Please fix the spacing as there is the white spacing on the left and right of the page and if you see the design then you will see the background is covering the whole width. please see the image below for reference.
- [ ] If you see the design then the text is aligned to the left but it doesn't pass the middle of the page. so I would suggest you please fix this issue as well. 😃 
### Figma design
figma
### Your design
his

File: index.html [hunk 2]
```diff
@@ -40,6 +51,7 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
           <h2>My Recent Works</h2>
           <hr>
         </hxr>
+
```
Comment (roshan-bajgain) [inline index.html:54]: - [ ] According to the project requirement you should stick to the design like color, font, margin, etc. please use `#091E42` color on your `h2` heading and there is a line that takes the rest of the page. please increase the width of the line. 😆 
### Your design
your
### Figma design
figma

File: index.html [hunk 3]
```diff
@@ -40,6 +51,7 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
...
         <section class="multi-stories ">
           <img src="assests/images/img_placeholder.png" alt="featured image">
```
Comment (roshan-bajgain) [inline index.html:57]: - [ ] Please align the first section of the image with the second section of the image on the same line. If you see the Figma design then you find the image on the same line. See the image below for reference
- [ ] Please use the border for the second image as well. that is: `professional image part`
### Figma design
figma
### Your design
his

File: index.html [hunk 4]
```diff
@@ -168,6 +181,85 @@ <h2>Profesional Art Printing Data</h2>
...
+            <p>A daily selection of privately personalized reads; 
+              no accounts or sign-ups required. 
+              has been the industry's standard</p>
+            
+            <hrx class="tags">
+              <ul>
+                <li>HTML</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+              </ul>
```
Comment (roshan-bajgain) [inline index.html:200]: - [ ] You should stick to the design as much as possible (e.g., font, colors, images, text, margins). please use the same color for text inside the buttons and also remove the white border
- [ ] Please implement for all the buttons
### Your design
your

### Figma design
fimga

File: index.html [hunk 5]
```diff
@@ -168,6 +181,85 @@ <h2>Profesional Art Printing Data</h2>
...
+            </hrx>
+          </hrx>
```
Comment (roshan-bajgain) [inline index.html:262]: - [ ] If you see the design below then you will find that the `About section is covering the whole width and there is a gap before the `About section. Please implement it. See image below for reference
### Your design
your

### Figma design
figma

File: index.html [hunk 6]
```diff
@@ -178,20 +270,23 @@ <h2 class="primary-text">About me</h2>
...
-            <li>CSS</li>
-          </ul>
-  
+      <section class="skills-container">
+        <section skills-category>
+          <hxr class="skils-title">
+            <img src="assests/images/diomand.png" alt="Diomand shape">
+            <h3 class="language">Language</h3>
+          </hxr>
+            <ul>
```
Comment (roshan-bajgain) [inline index.html:282]: - [ ] According to the project requirement you should stick to the design as much as possible (e.g., font, colors, images, text, margins). Please use the same color, if you see your design then the color is different in that 2 sections.
- [ ] Please remove the line as well there is 2 line
### Your design
his
### Figma design
figma

File: index.html [hunk 7]
```diff
@@ -215,13 +311,14 @@ <h3 class="language">Skills</h3>
...
+        </section>
       </section>
     </main>
   
     <footer> 
       <section class="contact">
         <p>I'm always interested in hearing about new projects,
-        so if you'd like to chat please get in</p>
+        so if you'd like to chat please get in touch</p>
         <form action="https://formspree.io/f/xjvzpjgq" method="post">
```
Comment (roshan-bajgain) [inline index.html:322]: - [ ] Please align the buttons and form at the same line and you should stick to the design as much as possible. Please see image below for reference
### Your design
his

### Figma design
fimga

File: index.html [hunk 8]
```diff
@@ -215,13 +311,14 @@ <h3 class="language">Skills</h3>
...
           <input type="text" name="fullname" placeholder="Full Name" maxlength="30" required>
           <input type="email" name="email" placeholder="Email address" required>
```
Comment (roshan-bajgain) [inline index.html:324]: - [ ] Please increase the width of the line and make it 100%. So it matches the design
### Your design
ssssssssssss

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @cherelemma , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] I cannot leave a comment there so I am leaving it here please implement it. please implement the `hover`, `Pressed`, and `Disable` for the buttons
### Your buttons on the hover
enable
### Your buttons on pressed
pressed
### Figma design
figma
- [ ] You design the mobile and desktop versions only and when I switch the design in between 375 and 768 px then I find the pages are not style. So to fix this issues you need to remove the media query for the mobile version then It works on every media query. Ie: In mobile.css remove `@media (max-width: 375px) {}` Then it will work
### Your design
yourrrrrrrrrrr

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jaflih — APPROVED** [review]
Hi @cherelemma   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

The Pull Request is following the GitHub Flow ✅
The README is well documented ✅
Linters are working properly ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @jaflih in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**